### ISSUE TITLE
hight info

### ISSUE THREAD

**Issue by arashroomi** [deepsense-ai/carla_birdeye_view#3]
I want to append another layer to` birdview ` to include the height of the objects. I prefer to have this with rendering mode disabled to speed up training. I already am working on adding the height of dynamic objects to the array. But regarding the height of the static objects such as walls, buildings, and parked cars apparently it's not that easy. Do you have any suggestions for a method to retrieve map heights data from CARLA?

**Comment by micmarty** [comment]
CARLA does not give such informations from API (only for vehicles). I guess it requires either using LIDAR to gather such data for each map and process it smartly, or... write C++ method for API that gets that info from Unreal assets. Both are tough IMHO.

**Comment by arashroomi** [comment]
Yes thanks. I have used a depth camera on top of the vehicle following the vehicel to do that. I  will integrate the data to the this library when its ready. 
The only problem is that the depth camera doesnt give orthogonal data of top-down view that is why I had to place the camera far from the vehicle and this causes problems when the vehicle passes under a bridge or so and it will lose the vision.

**Comment by micmarty** [comment]
Nice, feel free to submit PR any time :D
